lms/templates/pdf_viewer.html

```diff
@@ -412,12 +412,12 @@
   </div>
 </div>
     <script type="text/javascript" src="${static.url('/static/js/vendor/pdfjs/viewer.js')}"></script>
-<<<<<<< HEAD
-    <!-- TODO: fix PDF analytics
-        <script type="text/javascript" src="${static.url('/static/js/pdf-analytics.js')}"></script>
-    -->
-=======
+<!-- <<<<<<< HEAD -->
```

Comment:
@caesar2164 It looks like this is not commented out upstream; is that what we want to now?